Sources/ComposableArchitecture/TestStore.swift

```diff
@@ -488,13 +488,9 @@ public final class TestStore<State, Action> {
   public var timeout: UInt64
 
   private let file: StaticString
-  private var line: UInt
+  private let line: UInt
```

Comment:
We previously advanced the test store's line whenever it was interacted with so that failures for unasserted things occur toward the end of the test. This is actually a little confusing, though, as failure messages can be tied to assertion lines that passed just fine. So instead we can maintain a constant `line` for where the store was initialized, and emit failures there.